data/json/items/gun/faults_gun.json

```diff
@@ -35,6 +74,29 @@
       }
     ]
   },
+  {
+    "id": "fault_gun_unlubricated",
+    "type": "fault",
+    "name": "Unlubricated",
+    "description": "Either this gun is brand new and came without lubrication or it was recently cleaned with a solvent without oiling afterwards. Either way, it's not lubricated and will not cycle properly, and can even be damaged.",
```

Comment:
```suggestion
    "description": "Either this gun is brand new and came without lubrication or it was recently cleaned with a solvent without oiling afterwards.  Either way, it's not lubricated and will not cycle properly, and can even be damaged.",
```

Reply:
Double space after period? Isn't that an old artifact from the days of typewriters? https://style.mla.org/number-of-spaces-after-period/
Pretty sure it's generally supposed to be one space, unless there's some specific purpose or format in mind for some reason (can't imagine why).

Reply:
For general use that is true, however, it is this games convention that there are double spaces after one period. IIRC it has something to do with monospaced fonts.